lib/gigya_flutter_plugin.dart

```diff
@@ -219,4 +219,67 @@ class GigyaSdk {
   }) {
     return GigyaFlutterPluginPlatform.instance.sso(parameters: parameters);
   }
+
+  /// Check isLocked by biometrics.
```

Comment:
I have a few suggestions here:
1) Pass the parameters to the actual methods. Now they are unused.
2) You could apply the documentation updates from the implementation? They include phrasing adjustments, which would be handy here as well.

Reply:
Can you elaborate on the second point? I did not understand

Reply:
I had suggested documentation updates in `gigya_flutter_plugin_platform_interface.dart`. These documentation adjustments might also apply to this file.